addons/main_a3/CfgMods.hpp

```diff
@@ -2,15 +2,13 @@ class CfgMods {
     class PREFIX {
         author = "$STR_CBA_Author";
         dir = "@CBA_A3";
-        name = "Community Base Addons v3.9.1";
+        name = "Community Base Addons v0.0.0";
         picture = "x\cba\addons\main\logo_cba_ca.paa";
         hidePicture = 1;
         hideName = 1;
         actionName = "Website";
         action = "$STR_CBA_URL";
         description = "Bugtracker: https://github.com/CBATeam/CBA_A3/issues<br/>Documentation: https://github.com/CBATeam/CBA_A3/wiki";
-        logo = "logo_cba_ca.paa";
```

Comment:
This works if you have the logo inside a PBO as well, not just root of `@mod` folder.